### PR TITLE
Inheritance wallet - Part 4 - Recovery

### PR DESCRIPTION
**Pull request by rabbitholiness**
This is the last part of the inheritance wallet reference design. This PR adds a page about wallet recovery. It describes how Christina and David use their inheritance keys to recover their parents' wallet and access their inheritance. 

👀 Check the preview [here](https://deploy-preview-1096--bitcoin-design-site.netlify.app/guide/inheritance-wallet/wallet-recovery/) 👀

### CODE REVIEW

File: guide/inheritance-wallet/recovery.md [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+    - file: inheritance-transaction-sign/app-home-sending
+      alt: App home screen showing a pending transaction.
+      caption: The home page shows the pending transaction.
+---
+
+<!--
+
+Editor's notes
+
+This page covers a multi-key wallet that uses timelocks to provide additional recovery options.
```
Comment (GBKS) [inline guide/inheritance-wallet/recovery.md:92]: Update to match this page.

File: guide/inheritance-wallet/recovery.md [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+---
+
+<div class="glossary-toc" markdown="1">
+ * Table of contents
+{:toc}
+</div>
+
+---
+
+One day, a couple of years later, Christina and David get sad news that their parents, Alice and Bob, were involved in a deadly car accident. While dealing with the inheritance proceedings, Christina and David log in to their shared family vault in their password manager.
```
Comment (GBKS) [inline guide/inheritance-wallet/recovery.md:112]: > One day, a couple of years later

Later? I assume it's referencing the wallet setup. Maybe change it to `a couple of years after the family set up their inheritance wallet`?

File: guide/inheritance-wallet/recovery.md [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+
+{% include image-gallery.html pages = page.images_wallet-import %}
+
+However, there is a catch: they cannot spend the funds with their inheritance keys, because they have not been unlocked yet. Alice’s and Bob's last transaction was almost 5 months ago, which means that it will take more than 7 months until Christina and David can use their inheritance keys to move the funds. We have covered the wallet configuration in more detail on the [use case page]({{ '/guide/inheritance-wallet/use-case-and-scope/' | relative_url }}). Christina and David could circumvent this restriction, if they would have access to their parents’ signing devices. Unfortunately, they don’t, so they will need to wait. 
+
+### Distributing the inheritance
+
+After the timelock for the inheritance key set expires, Chrstina and David get together over dinner. They want to send half of the funds to Christina’s personal wallet and the other half to David’s wallet.
```
Comment (GBKS) [inline guide/inheritance-wallet/recovery.md:129]:
```suggestion
After the timelock for the inheritance key set expires, Christina and David get together over dinner. They want to send half of the funds to Christina’s personal wallet and the other half to David’s wallet.
```

File: guide/inheritance-wallet/recovery.md [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+
+{% include image-gallery.html pages = page.images_inheritance-transaction-create %}
+
+The transaction has been created and is ready to be signed. Christina goes ahead and sings it with her hardware signing device.
```
Comment (GBKS) [inline guide/inheritance-wallet/recovery.md:133]: Could this be a batch transaction? Would save fees and reduce the number of signing rounds.

File: guide/inheritance-wallet/recovery.md [hunk 5]
```diff
@@ -0,0 +1,151 @@
...
+
+{% include image-gallery.html pages = page.images_inheritance-transaction-sign-christina %} 
+
+After Christina is done, David goes on to sign the transaction with his device as well.
+
+{% include image-gallery.html pages = page.images_inheritance-transaction-sign-david %} 
+
+The first half of the inheritance has been sent to Christina's wallet. While the transaction is still being confirmed, David and Christina create and sign the second transaction, which will send the other half of the funds to David's wallet. 
+
```
Comment (GBKS) [inline guide/inheritance-wallet/recovery.md:142]: The funds are now distributed to the heirs, but the social & legal process is not finished. The lawyer may need to be informed, maybe they need to take screenshots of the fund distribution (like a receipt) for future reference. Maybe they also want to delete the wallet data, swipe the signing devices, or other stuff. Just made this up quickly, but that clean-up might be a good way to wrap up.

**Review by GBKS — CHANGES_REQUESTED** [review]
Looking good. Some minor feedback, and a request for adding more info to the wrap-up.

I am also wondering it the page title is right. Recovery sounds like something got lost. But what's being described is part of the planned inheritance process. Is there maybe a good context-specific term? Succession? What do you think?

**Comment by rabbitholiness** [comment]
I will open a new PR, as a I had technical issues with the branch.

**Pull request closed without merging**